### ISSUE TITLE
Fatal Python error: Segmentation fault on qutebrowser 1.13.0.r2.gcb15192cd-1

### ISSUE THREAD

**Issue by mnabila** [qutebrowser/qutebrowser#5707]
**Version info**:

> Please copy the first block from :version, not just the qutebrowser version

sorry unable show :version because when i try open qutebrowser from terminal or application launcher get this issue 
![Screenshot_20200908_070900](https://user-images.githubusercontent.com/37516576/92421346-4bfba380-f1a2-11ea-9ed8-828ceb244b17.png)

**Does the bug happen if you start with `--temp-basedir`?**:
nothing happend
![Screenshot_20200908_071013](https://user-images.githubusercontent.com/37516576/92421359-63d32780-f1a2-11ea-8368-5b0e5106d9d8.png)

**Description**
I had this problem starting last month, the steps I took so that qutebrowser could run normally was to downgrade the qt version that I was using. but some applications that use qt crashed so that the solution I took was no longer effective

**How to reproduce**

try install qutebrowser from AUR and official repository but issue not gone, try downgrade qt 5.15 to qt 5.14 but this not best solutions for me.

**ps: sorry if my english to bad**

**Comment by mnabila** [comment]
after following this tutorial about strace [A diagnostic, debugging and instructional userspace tracer
](https://www.tecmint.com/strace-commands-for-troubleshooting-and-debugging-linux/) i get this result.
![Screenshot_20200909_055952](https://user-images.githubusercontent.com/37516576/92536532-220ab580-f264-11ea-9980-2d3a80c3cd92.png)

now i thin get conclusion about this issue, issue from qt5ct not from qutebrowser or qt5.15. after that I tried to change QT_QPA_PLATFORMTHEME from qt5ct to gtk2 and finally this problem was solved.
![Screenshot_20200909_060111](https://user-images.githubusercontent.com/37516576/92536750-abba8300-f264-11ea-9b90-0c903a142689.png)
